opentelemetry-otlp/src/exporter/mod.rs

```diff
@@ -28,18 +28,29 @@ pub const OTEL_EXPORTER_OTLP_PROTOCOL: &str = "OTEL_EXPORTER_OTLP_PROTOCOL";
 /// Compression algorithm to use, defaults to none.
 pub const OTEL_EXPORTER_OTLP_COMPRESSION: &str = "OTEL_EXPORTER_OTLP_COMPRESSION";
 
-#[cfg(feature = "http-proto")]
+#[cfg(feature = "http-json")]
+/// Default protocol, using http-json.
+pub const OTEL_EXPORTER_OTLP_PROTOCOL_DEFAULT: &str = OTEL_EXPORTER_OTLP_PROTOCOL_HTTP_JSON;
```

Comment:
Will `http/json`  be the default if all or any two (including `http-json`) feature is enabled? If I remember correctly, http/protobuf was the default protocol in that case as per the specs.

Reply:
The spec says `SDK-dependent, but will typically be either http/protobuf or grpc`.  The feature `http-json` is not enabled in default feature list for `opentelemetry_otlp` crate. If someone is enabling it explicitly, it is reasonable to assume they want to go with `http/json` as the protocol imho. I'm open to change if that is not the case.

Reply:
Thanks. This is what I see in the [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/e6eccba97ebaffbbfad6d4358408a2cead0ec2df/specification/protocol/exporter.md#specify-protocol):

``
If no configuration is provided the default transport SHOULD be http/protobuf unless SDKs have good reasons to choose grpc as the default (e.g. for backward compatibility reasons when grpc was already the default in a stable SDK release).
``

This means the default enabled in `Cargo.toml` should actually be `http-proto`  and not [grpc-tonic](https://github.com/open-telemetry/opentelemetry-rust/blob/cc80ec6aa46cb42b61e935d5146ae58e115d3580/opentelemetry-otlp/Cargo.toml#L64).

Not related to this PR, but separate issue :)

Reply:
Ah. I must have referred to the doc instead https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_protocol

Agree, the spec requires it to be `http-proto` as default. Will open an issue and try to address it.

Reply:
Thanks @ramgdev